openvdb_houdini/houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -49,7 +51,6 @@
 namespace hvdb = openvdb_houdini;
 namespace hutil = houdini_utils;
 
-
```

Comment:
Some minor formatting changes here and there I'd recommend not including for a simpler diff...